script/mfa_required_for_user

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
Ashley and I initially worked on this as a script. However, I couldn't get it to run properly so I converted it over to a rake task.